js/start.js

```diff
@@ -253,27 +256,167 @@ function onboardIfNeeded() {
   return onboardIfNeededDeferred.promise();
 }
 
-// let's start our flow
-fetchConfig().done((data) => {
-  app.profile = new Profile({ id: data.guid });
-
-  app.settings = new Settings();
-  // We'll default our server language to whatever is stored locally.
-  app.settings.set('language', app.localSettings.get('language'));
-
-  // Beyond the start-up flow in this file, any language changes should ideally
-  // be done via a save on a clone of the app.settings model. When the save succeeds,
-  // update the app.settings model which will in turn update our local
-  // settings model. You shouldn't be directly updating the language in our local
-  // settings model.
-  app.settings.on('change:language', (settingsMd, lang) => {
-    app.localSettings.save('language', getValidLanguage(lang));
-  });
+ // let's start our flow - do we need onboarding?,
+ // fetching app wide models...
+function start() {
+  fetchConfig().done((data) => {
```

Comment:
Start.js is, IMO, getting a little too bulky. When I implement the real start-up / connection flow, I'll look into separating some things out.